src/impy/kinematics.py

```diff
@@ -80,22 +173,28 @@ def sort_list(k, idcs):
         self.component_Z = sort_list(self.component_Z, idcs)
         self.component_name = sort_list(self.component_name, idcs)
 
+    def get_maximum_AZ(self):
+        a_val = self.component_A
+        max_ind = a_val.index(max(a_val))
+        return self.component_A[max_ind], self.component_Z[max_ind]
+
+    def set_rng_seed(self, seed):
+        self.rng = self.default_rng(seed)
```

Comment:
The seed and the RNG should be only set in the constructor. Please remove.

Reply:
Yes, it is not needed. Removed